controllers/Sign_up.py

```diff
@@ -0,0 +1,106 @@
+from flask import request, jsonify
+from flask.views import MethodView
```

Comment:
This line is redundant, not used in the code

Reply:
I have removed `from flask.view import MethodView`.